railties/lib/rails/generators/rails/app/interactive_options.rb

```diff
@@ -0,0 +1,144 @@
+        end
+
+        def inquire_action_mailer
+          say "Action mailer is a framework for designing email services: https://guides.rubyonrails.org/action_mailer_basics.html"
+          inquire_boolean("Skip action mailer?", options[:skip_action_mailer])
+        end
+
+        def inquire_action_mailbox
+          say "Action Mailbox routes incoming emails to controller-like mailboxes for processing in Rails: https://guides.rubyonrails.org/action_mailbox_basics.html"
+          inquire_boolean("Skip action mailbox?", options[:skip_action_mailbox])
+        end
+
+        def inquire_active_storage
+          say "Active Storage makes it simple to upload and reference files in cloud services: https://guides.rubyonrails.org/active_storage_overview.html"
+          inquire_boolean("Skip active storage?", options[:skip_active_storage])
+        end
+
+        def inquire_action_text
+          say "Action Text brings rich text content and editing to Rails: https://guides.rubyonrails.org/action_text_overview.html"
+          inquire_boolean("Skip action text?", options[:skip_action_text])
+        end
+
+        def inquire_bootsnap
+          say "Bootsnap makes Rails application boot faster: https://github.com/Shopify/bootsnap"
+          inquire_boolean("Skip bootsnap?", options[:skip_bootsnap])
+        end
+
+        def inquire_jbuilder
+          say "Jbuilder generates JSON objects with a Builder-style DSL: https://github.com/rails/jbuilder"
+          inquire_boolean("Skip jbuilder?", options[:skip_jbuilder])
+        end
+
+        def inquire_javascript
+          say "Use yarn to handle your package dependency management."
+          inquire_boolean("Skip javascript?", options[:skip_javascript])
+        end
+
+        def inquire_spring
+          say "Rails application preloader, speeds up development by keeping your application running in the background: https://github.com/rails/spring"
+          inquire_boolean("Skip spring?", options[:skip_spring])
+        end
+
+        def inquire_sprockets
+          say "Sprockets is a Ruby library for compiling and serving web assets: https://github.com/rails/sprockets"
+          inquire_boolean("Skip sprocket?", options[:skip_sprockets])
+        end
+
+        def inquire_system_tests
+          say "System tests are useful for writing end-to-end tests with a tool like capybara."
+          inquire_boolean("Skip system tests?", options[:skip_system_tests])
+        end
+
+        def inquire_turbolinks
+          say "Turbolinks makes navigating your web application faster: https://github.com/turbolinks/turbolinks"
+          inquire_boolean("Skip turbolinks?", options[:skip_turbolinks])
+        end
+
+        def inquire_webpack
+          say "Webpacker manages your app-like JavaScript modules in Rails: https://github.com/rails/webpacker"
+          inquire_boolean("Skip webpack?", options[:skip_webpack_install])
+        end
+
+        def inquire_webpack_library
```

Comment:
Since webpack isn't the default anymore in Rails 7 let's revisit this phrasing/UX to address that. 

Maybe two tier query asking if this is something they'd want. Then what they want